### PR TITLE
Update wrapper.h

### PR DESCRIPTION
**Pull request by klebz**
including these headers seems prerequisite for accessing parts of the libevent api.  

In my case, I needed access to the libevent http hooks from rust.

### CODE REVIEW

File: libevent-sys/wrapper.h [hunk 1]
```diff
@@ -1,5 +1,15 @@
 #include <event.h>
 
+#include <event2/event.h>       // The primary libevent header
+#include <event2/thread.h>      // Functions for use by multithreaded programs
```
Comment (jmagnuson) [inline libevent-sys/wrapper.h:4]: Threading support is already included in the wrapper:
https://github.com/jmagnuson/libevent-rs/blob/27cc8f1f064c88125d6578b2b426c135308dd54d/libevent-sys/wrapper.h#L7-L9

and should only require the `threading` cargo feature flag, which is enabled by default.

**Comment by tklebanoff** [comment]
I think we ought to be able to use the `evhttp*` functionality without a fork.

what do you think? is it agreeable to merge these changes?

**Comment by jmagnuson** [comment]
Correct me if I'm wrong; I think the `buffer*` related headers could be removed since they're already [included](https://github.com/libevent/libevent/blob/release-2.1.12-stable/include/event.h#L71-L73) in `event.h`. But yeah the others should definitely get included.

**Comment by klebz** [comment]
good point! this sounds reasonable!  I will modify

**Comment by klebz** [comment]
i am wondering: is there anything I ought/need to do on my end to prepare this for merge? thanks for the help!

**Review by jmagnuson — APPROVED** [review]
LGTM. I'll do some further testing before cutting a release, but this is good to merge. Thanks!

**Comment by kleb6** [comment]
thanks! -- what do you think about initiating a crates.io release for these changes?

**Pull request merged**